styleguide-app/assets/TextArea.js

```diff
@@ -43,7 +43,7 @@ Object.defineProperties(AutoresizeTextArea.prototype, {
   }
 });
 
-AutoresizeTextArea.prototype._onInput = function() {
+AutoresizeTextArea.prototype._resize = function() {
```

Comment:
Thanks for pulling in these changes from the `TextArea.js` files from the monolith.